experimental/casemapping/tests/gen_greek_to_me.rs

```diff
@@ -0,0 +1,146 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use databake::Bake;
+use icu_casemap::greek_to_me::{self, GreekPrecomposedLetterData};
+use icu_casemap::CaseMapper;
+use icu_codepointtrie_builder::{CodePointTrieBuilder, CodePointTrieBuilderData};
+use icu_collections::codepointtrie::{CodePointTrie, TrieType};
+use icu_locid::langid;
+use icu_normalizer::DecomposingNormalizer;
+use icu_properties::{maps, GeneralCategoryGroup, Script};
+use std::io::Write;
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+use std::{env, fs, str};
+
+fn main() {
+    let size = u32::from(char::MAX);
+    let mut vec = Vec::with_capacity(size as usize);
+    let mut encode_scratch = [0; 4];
+
+    let decomposer = DecomposingNormalizer::new_nfd();
+    let script = maps::script();
+    let gc = maps::general_category();
+    let cm = CaseMapper::new();
+    let und = langid!("und");
+
+    for ch in 0..=size {
```

Comment:
nit: you can save some boilerplate by doing
```rust
let vec = (0..'\u{D7FF}').chain('\u{E000}'..char::MAX).map(|ch| {
  ...
}).collect();

Reply:
Nope, this _must_ iterate over all values between the max including surrogates because the vector is indexed by code point

Reply:
Still you can collect into a vec instead of manually reserving and pushing.

Reply:
I don't really like having massive `.map()`s

Reply:
I really don't like having mutable `Vec`s :)